notebooks/vertex_genai/solutions/deployment_adk_agent.ipynb

```diff
@@ -0,0 +1,1061 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        print(f"\n--- [Call from: {author}] ---")
</pre><p>I would say: </p>

<pre class="ql-syntax" spellcheck="false">print(f"\n--- [Event: {author}] ---")
</pre><p>since each iteration represent a Event.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>